ui/src/js/profile/ToolsPage.js

```diff
@@ -0,0 +1,48 @@
+import React, {Component} from 'react';
+import {render} from 'react-dom';
+import Boostrap from 'bootstrap';
+import axios from 'axios';
+
+import Tools from './tools/Tools';
+
+export default class ToolsPage extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      user: {
+      },
+      pageCount: 0
+    };
+  }
+
+  callNextPage() {
+    var newPage = this.state.pageCount + 1;
+    this.callPage(newPage);
+    this.setState({pageCount: newPage});
+  }
+
+  callPreviousPage() {
+    var newPage = this.state.pageCount - 1;
+    this.callPage(newPage);
+    this.setState({pageCount: newPage});
+  }
+
+  updateLanguage(value) {
+    this.refs['navbar'].updateLanguage(value);
+  }
+
```

Comment:
this method seems to be a copy&paste mistake, i don't see any option to call this.

Reply:
You are right, there is no functionality to switch the language in the component.